src/Toggl2Vertec/Properties/launchSettings.json

```diff
@@ -3,6 +3,18 @@
     "Toggl2Vertec": {
       "commandName": "Project",
       "commandLineArgs": "clear --debug"
+    },
+    "check": {
```

Comment:
Now I know who uses _that_ functionality.

Reply:
rider was doing it's best to steer me in that direction _shrug_...